sdk/eventhub/azure-eventhubs/examples/send.py

```diff
@@ -13,44 +13,23 @@
 
 # pylint: disable=C0111
 
-import logging
 import time
 import os
-
 from azure.eventhub import EventHubClient, EventData, EventHubSharedKeyCredential
 
-import examples
-logger = examples.get_logger(logging.INFO)
-
-
-HOSTNAME = os.environ.get('EVENT_HUB_HOSTNAME')  # <mynamespace>.servicebus.windows.net
-EVENT_HUB = os.environ.get('EVENT_HUB_NAME')
-
-USER = os.environ.get('EVENT_HUB_SAS_POLICY')
-KEY = os.environ.get('EVENT_HUB_SAS_KEY')
-
-try:
-    if not HOSTNAME:
-        raise ValueError("No EventHubs URL supplied.")
-
-    client = EventHubClient(host=HOSTNAME, event_hub_path=EVENT_HUB, credential=EventHubSharedKeyCredential(USER, KEY),
-                            network_tracing=False)
-    producer = client.create_producer(partition_id="0")
-
-    try:
-        start_time = time.time()
-        with producer:
-            # not performance optimal, but works. Please do send events in batch to get much better performance.
-            for i in range(100):
-                ed = EventData("msg")
-                logger.info("Sending message: {}".format(i))
-                producer.send(ed)
-    except:
-        raise
-    finally:
-        end_time = time.time()
-        run_time = end_time - start_time
-        logger.info("Runtime: {} seconds".format(run_time))
 
-except KeyboardInterrupt:
-    pass
+HOSTNAME = os.environ['EVENT_HUB_HOSTNAME']  # <mynamespace>.servicebus.windows.net
+EVENT_HUB = os.environ['EVENT_HUB_NAME']
+USER = os.environ['EVENT_HUB_SAS_POLICY']
+KEY = os.environ['EVENT_HUB_SAS_KEY']
+
+client = EventHubClient(host=HOSTNAME, event_hub_path=EVENT_HUB, credential=EventHubSharedKeyCredential(USER, KEY),
+                        network_tracing=False)
+producer = client.create_producer(partition_id="0")
+start_time = time.time()
```

Comment:
this value is never used, should either be removed, or missing runtime prints restored

Reply:
thanks, added runtime prints